docs/integrations/webhooks/inngest.mdx

```diff
@@ -0,0 +1,215 @@
+
+The code below creates another function using the same `clerk/user.created` event and adds the logic to send the welcome email:
+
+
+```typescript
+const sendWelcomeEmail = inngest.createFunction(
+ { id: 'send-welcome-email' },
+ { event: 'clerk/user.created' },
+ async ({ event }) => {
+   const { user } = event.data
+   const { first_name } = user
+   const email = user.email_addresses.find(e =>
+     e.id === user.primary_email_address_id
+   ).email
+   await emails.sendWelcomeEmail({ email, first_name })
+ }
+)
+```
+
+
+Now, you have a function that utilizes the same Clerk webhook event for another purpose. Clerk webhook events can be used for all sorts of application lifecycle use cases. For example, adding users to a marketing email list, starting a Stripe trial, or provisioning new account resources.
+
+
+### Sending a delayed follow-up email
+
+Every Inngest function handler has an additional `step` object which provides tools to create more complex functions. Using `step.run` allows you to encapsulate specific code that will be automatically retried ensuring that issues with one part of your function don't force the entire function to re-run. Additionally, [other tools like `step.sleep`](https://www.inngest.com/docs/reference/functions/step-sleep), are available to extend functionality.
+
+The code below sends a welcome email, then uses `step.sleep` to wait for three days before sending another email offering a free trial:
+
+```typescript
+const sendOnboardingEmails = inngest.createFunction(
+ { id: 'onboarding-emails' },
+ { event: 'clerk/user.created' },
+ async ({ event, step }) => { // ← step is available in the handler's arguments
+   const { user } = event.data
+   const { first_name } = user
+   const email = user.email_addresses.find(e =>
+     e.id === user.primary_email_address_id
+   ).email
+
+   await step.run('welcome-email', async () => {
+     await emails.sendWelcomeEmail({ email, first_name })
+   })
+
+   // wait 3 days before second email
+   await step.sleep('wait-3-days', '3 days')
+
+
+   await step.run('trial-offer-email', async () => {
+     await emails.sendTrialOfferEmail({ email, first_name })
+   })
+ }
+)
+```
+
+Now, you've extended the usefulness of Clerk webhook events even further to build an onboarding drip email campaign in just a few lines of code.
+
+
+## Testing webhook events using the Inngest Dev Server
+
+During local development with Inngest, you can use the [Inngest Dev Server](https://www.inngest.com/docs/local-development) to run and test your functions on your own machine. To start the server, in your project directory run the following command:
+
+```sh filename="terminal"
+npx inngest-cli@latest dev
+```
+
+In your browser open [http://localhost:8288](http://localhost:8288) to see the development UI.
+
+To quickly get events to test within Dev Server, you can select any individual event from the **Events** tab then select the **Send to Dev Server**.
```

Comment:
It wasn't super clear here that this step was back in the Inngest dashboard.

Reply:
I've just pushed a commit to the previous paragraph. Do you think it clarifies that we are now operating on the Dev Server UI? (It's not the Inngest Dashboard, it's local development)

Reply:
This definitely better. I was confused -- I ran the dev server but didn't really do anything with it. It looked different on its starting screen so I thought the following step was back in the Dashboard. Great clarification for someone just reading!

Please resolve once you read this.